lib/mrsk/configuration/accessory.rb

```diff
@@ -23,14 +23,20 @@ def hosts
     hosts_from_host || hosts_from_hosts || hosts_from_roles
   end
 
-  def port
-    if port = specifics["port"]&.to_s
+  def ports
+    if (specifics.keys & ["port", "ports"]).size > 1
+      raise ArgumentError, "Specify one of `port` or `ports` for accessory `#{name}`"
+    end
+
+    (ports_from_port || ports_from_ports)&.map do |p|
+      port = p.to_s
+
       port.include?(":") ? port : "#{port}:#{port}"
     end
```

Comment:
1st, nice, first time seeing the `array & array` syntax and I most admit I love it

But, I think that in this case `specifics['port'] && specifics['ports']` would be easier to read (just IMO?), same goes for the map leter?

```suggestion
    if specifics["port"] && specifics["ports"]
      raise ArgumentError, "Specify one of `port` or `ports` for accessory `#{name}`"
    end
    
    return ports_from_port if specifics["port"]
    return ports_from_ports if specifics["ports"]
```

And then later do conversion to `num:num`
Maybe even make the second return implicit and remove the if

Reply:
Thanks for review.

Didn't get this part though:

> And then later do conversion to num:num


By saying "later" you mean the callsite of `#ports`?

Reply:
> What do you think about adding another test to test if num:num is also mapped correctly (I know it works but for the sake of thoroughness)?

Not sure whether it is needed, this case is implicitly covered multiple times. The only thing that I didn't see - tests for the format correctness (i.e. to prevent typos in config like `8080: 8080` - with space or other letter).

Reply:
> By saying "later" you mean the callsite of #ports?

Yeah I meant to instead of doing ports_from_port and ports_from_ports and then formatting here, to do the formatting inside of those methods (or wrap here) but not to the map since it looks like more things are happening

> Multiple tests

Yeah true, my thoughts were more we test for `port: 1234:4321 / 4123` but only `ports: 1234`, but it's fine since we cover it by the other one

Reply:
Hi @Zeko369 ,

I've applied your change and proposal. Could you please take a look?